backend/utils/products.js

```diff
@@ -95,11 +123,32 @@ function appendToProductsFile(shop, productData) {
     }
   }
 
-  allProducts[existingIndex] = pick(productData, minimalistProductFields)
+  const hasVariants =
```

Comment:
supernit: can also use the Boolean constructor:
const hasVariants = Boolean(productData.variants && productData.variants.length)